server/sonar-server/src/main/java/org/sonar/server/search/Result.java

```diff
@@ -75,31 +72,52 @@ public Result(@Nullable BaseIndex<K, ?, ?> index, SearchResponse response) {
 
   private void processAggregation(Aggregation aggregation) {
```

Comment:
And separating aggregation stuff will allow to have Unit Test on it, as for the moment there's not UT on Result.